packages/server/src/admin/invite.ts

```diff
@@ -95,7 +99,11 @@ export async function inviteUser(
   const membership = await createProjectMembership(user, project, profile, request.accessPolicy);
 
   if (request.email && request.sendEmail !== false) {
-    await sendInviteEmail(request, user, existingUser, passwordResetUrl);
+    try {
+      await sendInviteEmail(request, user, existingUser, passwordResetUrl);
+    } catch (err) {
+      throw new Error('Could not send email. Make sure you have AWS SES set up.');
```

Comment:
@codyebberson how about line 45 - 53?

```
    const { membership } = await inviteUser({
      ...req.body,
      project: res.locals.project,
    });
    res.status(200).json(membership);
  } catch (err: any) {
    logger.info(err);
    res.status(200).json({ error: err });
  }
```

Since we agreed on attaining a status of 200, is it ok I keep the json body like this if it catches an error?